magicbot/magicrobot.py

```diff
@@ -540,6 +592,11 @@ def _inject(self, n, inject_type, cname, component):
     #        for f in d.keys():
     #            d[f] = f(component)     
 
+    def _update_feedback(self):
+        for (component, cname, name) in self._feedbacks:
+            # Put ntvalue at /robot/components/component/key
+            self.__nt.putValue('/components/{0}/{1}'.format(cname, getattr(component, name).__key__), getattr(component, name)())
```

Comment:
Maybe avoid calling `getattr` with the same arguments multiple times. `getattr(...)()` looks particularly ugly.